core/src/main/java/org/kiwix/kiwixmobile/core/search/SearchActivity.java

```diff
@@ -131,7 +128,7 @@ public void finish() {
     }
   }
 
-  @Override
+  @RequiresApi(api = Build.VERSION_CODES.M) @Override
```

Comment:
I doubt this requires API M

Reply:
I don't know why but if I don't do it android studio keeps on showing me error.

Reply:
I'll clone your branch and check it out

Reply:
Regardless you should never annotate a method you are not responsible for calling with this annotation
![image](https://user-images.githubusercontent.com/3358282/70067221-dc8acf00-15e5-11ea-925a-01e74a431be2.png)

The reason is explained quite well by AS:
![image](https://user-images.githubusercontent.com/3358282/70067422-2a073c00-15e6-11ea-9bd7-5c0e2f62d751.png)

I assume you hit `alt+enter` and got this:
![image](https://user-images.githubusercontent.com/3358282/70067504-473c0a80-15e6-11ea-9fa2-5b42fc69bdd7.png)
and then went for the first option.

What you wanted was the 2nd action but this reporting is very strange for a few reasons
1. It is only reporting the 2nd usage
1. We have 150,000 users and have never seen this crash

The underlying reason is that this is a broadcast from the android system, read more [here](https://medium.com/androiddevelopers/custom-text-selection-actions-with-action-process-text-191f792d2999), and will only happen on M and up anyways.

In summary not really a problem

Reply:
Oh! Got it.